CONTRIBUTING.md

```diff
@@ -84,7 +84,8 @@ Push the changes to the forked repository using:
 ```
 git push origin <Your-branch-name>
 ```
-Navigate to the original repository and make a pull request.
+Navigate to the original for of this repository and click on the branch you had commited to.
```

Comment:
Original For of this respository? Not sure what that means. Also committed is spelt wrong.